mce-capi-webhook-config/deployments/mutatingwebhook.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.beta.openshift.io/serving-cert-secret-name: mce-capi-webhook-config-service-cert
+  name: mce-capi-webhook-config-service
+  labels:
+    app: mce-capi-webhook-config
+spec:
+  ports:
+  - port: 9443
+    targetPort: 9443
+  selector:
+    app: mce-capi-webhook-config
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: mce-capi-webhook-config-configuration
+  annotations:
+    service.beta.openshift.io/inject-cabundle: "true"
+webhooks:
+- admissionReviewVersions:
+  - v1
+  - v1beta1
+  clientConfig:
+    caBundle: Cg==
+    service:
+      name: mce-capi-webhook-config-service
+      namespace: "default"
+      path: /mutate
+      port: 9443
+  failurePolicy: Fail
+  sideEffects: None
+  name: mce-capi-webhook-config.x-k8s.io
+  rules:
+  - apiGroups:
+    - cluster.x-k8s.io
+    - ipam.cluster.x-k8s.io
+    - runtime.cluster.x-k8s.io
```

Comment:
The apiVersion is different for runtime.cluster.x-k8s.io check [here](https://github.com/openshift/cluster-api/blob/master/config/webhook/manifests.yaml#L380)